### PR TITLE
Use \theoremstyle to avoid repeating \normalfont everywhere

### PR DESCRIPTION
**Pull request by meithecatte**

### CODE REVIEW

File: deciders/correctness-deciders.tex [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
 
-
+\theoremstyle{definition} % don't use italics
 \newtheorem{theorem}{Theorem}
 \newtheorem{definition}[theorem]{Definition}
 \newtheorem{lemma}[theorem]{Lemma}
 \newtheorem{proposition}[theorem]{Proposition}
 \newtheorem{corollary}[theorem]{Corollary}
 
+\theoremstyle{remark} % emphasize the "Remark N." with italics, not bold
```
Comment (meithecatte) [inline deciders/correctness-deciders.tex:34]: Inspired by the existence of `theoremstyle`, I changed the formatting of the remark-like environments – now they use italics for the part that theorems and definitions have bolded. That is arguably a more subjective matter. Apart from that, the PDF should render the same.

Comment (tcosmo) [reply deciders/correctness-deciders.tex:34]: On the side of subjective styling things, some complained about my convention of not using italics in theorem statements like most people do (I just find it hard to read but I acknowledge that a majority of scientists use that style).

Comment (meithecatte) [reply deciders/correctness-deciders.tex:34]: Yeah, I also dislike that style, and like how the document deviates from it. The common convention makes the inline equations much less visually distinct, and `\emph` looks weird when you need it.

Comment (tcosmo) [reply deciders/correctness-deciders.tex:34]: Ok ahah, I think we can stick to this style (anyway we might have to comply to some other style for conference/journal submissions).

However, a point that was brought up by savask and that we should probably change in the current paper is the number system for def/theorems/remarks/examples/etc and have a counter per section VS a unique counter in the document. But that can be handled in another PR.

**Pull request merged**